storage/range.go

```diff
@@ -590,24 +654,23 @@ func (r *Range) addReadWriteCmd(args proto.Request, reply proto.Response, wait b
 	// TODO(bdarnell): In certain raft failover scenarios, proposed
 	// commands may be abandoned. We need to re-propose the command
 	// if too much time passes with no response on the done channel.
-	raftChan := r.rm.ProposeRaftCommand(idKey, raftCmd)
+	errChan := r.rm.ProposeRaftCommand(idKey, raftCmd)
 
 	// Create a completion func for mandatory cleanups which we either
 	// run synchronously if we're waiting or in a goroutine otherwise.
 	completionFunc := func() error {
 		// First wait for raft to commit or abort the command.
 		var err error
-		if err = <-raftChan; err == nil {
+		if err = <-errChan; err == nil {
 			// Next if the command was commited, wait for the range to apply it.
 			err = <-pendingCmd.done
 		}
-
 		// As for reads, update timestamp cache with the timestamp
 		// of this write on success. This ensures a strictly higher
 		// timestamp for successive writes to the same key or key range.
 		r.Lock()
 		if err == nil && usesTimestampCache(args) {
-			r.tsCache.Add(header.Key, header.EndKey, header.Timestamp, txnMD5, false /* !readOnly */)
+			r.tsCache.Add(header.Key, header.EndKey, header.Timestamp, txnMD5, false)
```

Comment:
Restored...this was from back to back changes which moved from false to a variable named `readOnly`, then back to false with comment lost in the churn.